gateway/dataimpl/server_v1/queryserver.go

```diff
@@ -85,6 +89,14 @@ func (s *QueryServer) Query(in *query_v1.QueryRequest, out query_v1.QueryService
 		opts.QueryContext = fmt.Sprintf("`%s`.`%s`", in.GetBucketName(), in.GetScopeName())
 	}
 
+	if in.DurabilityLevel != nil {
+		durabilityLevel, errSt := durabilityLevelToCbqueryx(*in.DurabilityLevel)
```

Comment:
Could remove `durabilityLevel` and assign directly to `opts.DurabilityLevel`

Reply:
Can't due to it needing errSt assigned as well, and I prefer to avoid performing assignments from errored pre-operations.